guides/v2.2/cloud/configure/setup-cron-jobs.md

```diff
@@ -67,61 +82,52 @@ The following is an example cron job:
 
-We include a default cron job for Magento in the default file:
+1. Edit `.magento.app.yaml` in the root directory of the Magento code in the Git branch.
 
-```yaml
-crons:
-    cronrun:
-        spec: "* * * * *"
-        cmd: "php bin/magento cron:run"
-```
+1. Locate the `crons` section in the file and add your custom cron code.
 
-1.  Edit `.magento.app.yaml` in the root directory of the Magento code in the Git branch.
-2.  Locate the `crons` section in the file and add your custom cron code.
-
-    For example, you could add a reindexer cron job to run every three hours, 20 minutes after the hour (such as 12:20 am, 3:20 am, and so on):
+    For example, you could add a custom cron job to export the product catalog that runs every eight hours, 20 minutes after the hour (such as 12:20 am, 8:20 am, and so on):
 
     ```yaml
     crons:
         magento:
             spec: '* * * * *'
             cmd: 'php bin/magento cron:run'
         productcatalog:
-            spec: '20 */3 * * *'
-            cmd: 'bin/magento indexer:reindex catalog_product_category'
+            spec: '20 */8 * * *'
+            cmd: 'bin/magento export:start catalog_product_category'
     ```
-4.  Save the file and push updates to the Git branch.
 
-## Add cron jobs to environments {#env}
+1. Save the file and push updates to the Git branch to redeploy your Cloud project with the latest cron configuration settings.
+
+{:.bs-callout .bs-callout-info}
+If you discover that configuration changes are not applied to Staging and Production sites after you redeploy and do not see any related error messages in the log, submit a Support ticket that describes the attempted configuration changes. Include any updated YAML configuration files in the ticket.
+
+## Update custom cron jobs {#update}
 
-When you push the code, the cron jobs will be added to and run in the following environments:
+To add, remove, or update a custom cron job, change the configuration in the `crons` section of the `.magento.app.yaml` file for the Integration environment. Then, test the updates in the Integration environment before pushing the changes to the Production and Staging environments.
 
-*   Starter: All environments you push to including `Master`
-*   Pro: Only Integration environments you push to including `Master`
+## Troubleshooting cron jobs
 
-To add the cron jobs to Pro plan Staging and Production, you must [enter a ticket to Support]({{ page.baseurl }}/cloud/trouble/trouble.html). Request to have the cron jobs in `.magento.app.yaml` added to those environments. We recommend pushing the updates through to the Integration `master` branch.
+Magento has updated the ece-tools package to optimize cron processing on the Cloud platform and to fix cron-related issues. If you are having problems with cron processing, make sure that your project is using the most current version of the ece-tools package. See [Upgrades and patches]({{ page.baseurl }}/cloud/project/project-upgrade-parent.html).
```

Comment:
Use variables where possible:

```suggestion
Magento has updated the `{{site.data.var.ct}}` package to optimize cron processing on the {{site.data.var.ece}} platform and to fix cron-related issues. If you are having problems with cron processing, make sure that your project is using the most current version of the ece-tools package. See [Upgrades and patches]({{ page.baseurl }}/cloud/project/project-upgrade-parent.html).
```

Reply:
I think we'll need to update the way we refer to the Cloud product/project name to reflect new branding guidelines, which will be easier to do as a Global change.  I had forgotten about the ece-tools variable.